rplidar.go

```diff
@@ -249,86 +286,83 @@ func (d *Device) filterParams() (minAngleDiff float64, maxDistDiff float64) {
 }
 
 // Bounds returns the square meter bounds of the device.
-func (d *Device) Bounds(ctx context.Context) (r2.Point, error) {
```

Comment:
bounds is potentially not needed anymore (but still might be useful). It was a class function of lidar but no longer exists in camera

Reply:
I'll keep it around for now. Might remove it in a future iteration of cleaning up rplidar, with help of Eric.